### PR TITLE
fix(DEV-12336): fix table filters layout and make them responsive

### PR DESCRIPTION
**Pull request by chebum**
This PR fixes layout of table filters

### CODE REVIEW

File: packages/sdk-themes/src/themes/monite.ts [hunk 1]
```diff
@@ -318,6 +326,26 @@ export const defaultMoniteComponents: Components<Omit<Theme, 'components'>> = {
...
+          width: '100%',
+        },
+      },
+    },
+  },
+  MuiStack: {
+    styleOverrides: {
+      root: {
+        '&.Monite-Filters': {
+          '& > *': {
```
Comment (chebum) [inline packages/sdk-themes/src/themes/monite.ts:340]: @radist2s I made this changes in theme itself, but the filters will look worse in the blank Material UI theme. I can move the layout styles to the sdk itself.

File: packages/sdk-react/src/components/approvalRequests/ApprovalRequestsTable/ApprovalRequestsFilter/ApprovalRequestsFilter.tsx [hunk 2]
```diff
@@ -31,86 +31,78 @@ interface FilterProps {
 export const ApprovalRequestsFilter = ({ onChangeFilter }: FilterProps) => {
   const { i18n } = useLingui();
   const { root } = useRootElements();
+  const className = 'Monite-ApprovalRequestsFilters';
 
   return (
-    <Grid container justifyContent="space-between">
```
Comment (radist2s) [inline packages/sdk-react/src/components/approvalRequests/ApprovalRequestsTable/ApprovalRequestsFilter/ApprovalRequestsFilter.tsx:36]: The same advice: would be better using just css to pull elements to the right.

File: packages/sdk-react/src/components/counterparts/CounterpartsTable/Filters/Filters.tsx [hunk 3]
```diff
@@ -32,95 +25,89 @@ export const Filters = ({ onChangeFilter, showCategories }: Props) => {
...
+          <InputLabel id="category">{t(i18n)`Category`}</InputLabel>
           <Select
-            labelId="type"
-            label={t(i18n)`Type`}
+            labelId="category"
+            label={t(i18n)`Category`}
             defaultValue="all"
             MenuProps={{ container: root }}
             onChange={(search) => {
-              onChangeFilter(FILTER_TYPE_TYPE, search.target.value);
```
Comment (radist2s) [inline packages/sdk-react/src/components/counterparts/CounterpartsTable/Filters/Filters.tsx:61]: As I see, this refactoring with the selectors `onChangeFilter` and `value`s should be a part for separate PR. Moreover, this refactoring the code without need adds difficulties for other developers on merges.
Btw, I would not change options to `'false' | 'true'`

**Review by radist2s — CHANGES_REQUESTED** [review]

File: packages/sdk-react/src/components/misc/FilterContainer.tsx [hunk 4]
```diff
@@ -0,0 +1,53 @@
+import { ReactNode } from 'react';
+
+import { classNames } from '@/utils/css-utils';
+import { Stack } from '@mui/material';
+
+interface Props {
```
Comment (vkuprin) [inline packages/sdk-react/src/components/misc/FilterContainer.tsx:6]: We have a convention to name them as `ComponentNameProps`, so in this case `FilterContainerProps`

**Review by vkuprin — APPROVED** [review]

**Pull request merged**